src/main/java/gov/cms/madie/terminology/service/FhirTerminologyService.java

```diff
@@ -0,0 +1,41 @@
+package gov.cms.madie.terminology.service;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.parser.IParser;
+import gov.cms.madie.models.measure.ManifestExpansion;
+import gov.cms.madie.terminology.models.UmlsUser;
+import gov.cms.madie.terminology.webclient.FhirTerminologyServiceWebClient;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.hl7.fhir.r4.model.Bundle;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class FhirTerminologyService {
+  FhirContext ctx = FhirContext.forR4();
```

Comment:
there is already a FHIRContext available in service. Can be injected it here.

Reply:
Yeah I have seen that, But I am not sure why we have to go through an extra step of having a new Bean for FhirContext when we can directly access forR4() which is a static method.

Reply:
that is because as per FHIRCOntext documentation it is lil bit heavy operation

> The FHIR context is the central starting point for the use of the HAPI FHIR API. It should be created once, and then used as a factory for various other types of objects (parsers, clients, etc.).

Reply:
Cool, Good to know, and I updated it to inject the bean.